man/docker-build.1.md

```diff
@@ -229,18 +248,19 @@ as context. The Dockerfile at the root of the repository is used as
 Dockerfile. This only works if the GitHub repository is a dedicated
 repository.
 
-    docker build github.com/scollier/Fedora-Dockerfiles/tree/master/apache
+    docker build github.com/scollier/Fedora-Dockerfiles/tree/master/apache .
```

Comment:
I'm looking for alternative now..looks like this hasn't been working for awhile :)  
http://stackoverflow.com/questions/27683927/docker-how-to-build-an-image-from-a-non-master-branch-on-github-repository   
thanks for the fb, i'm on it...